pkg/crawler/collector.go

```diff
@@ -41,7 +41,7 @@ func Collector(ctx context.Context, url string, projectPath string, collyOpts ..
 	c.OnHTML("img[src]", func(e *colly.HTMLElement) {
 		// src attribute
 		link := e.Attr("src")
-		if strings.Contains(link, "data:image") {
+		if strings.HasPrefix(link, "data:image") {
```

Comment:
Was there any benefit using `HasPrefix` over `Contains`?

Reply:
Hi, i think is more efficient beacause Contains is going to check in all string and HasPrefix only firsta characters(Fail-fast).
From [rfc](https://datatracker.ietf.org/doc/html/rfc2397) this uris always strart with this patron `data:[media]`

Reply:
Makes sense nice catch! Feel free to merge at any point.